pkg/config/config.go

```diff
@@ -38,15 +38,16 @@ var (
 )
 
 type Options struct {
-	Owner        string `mapstructure:"owner"`
-	GitRepo      string `mapstructure:"git-repo"`
-	ChartsRepo   string `mapstructure:"charts-repo"`
-	IndexPath    string `mapstructure:"index-path"`
-	PackagePath  string `mapstructure:"package-path"`
-	Token        string `mapstructure:"token"`
-	GitBaseURL   string `mapstructure:"git-base-url"`
-	GitUploadURL string `mapstructure:"git-upload-url"`
-	Commit       string `mapstructure:"commit"`
+	Owner           string `mapstructure:"owner"`
+	GitRepo         string `mapstructure:"git-repo"`
+	ChartsRepo      string `mapstructure:"charts-repo"`
+	IndexPath       string `mapstructure:"index-path"`
+	PackagePath     string `mapstructure:"package-path"`
+	Token           string `mapstructure:"token"`
+	GitBaseURL      string `mapstructure:"git-base-url"`
+	GitUploadURL    string `mapstructure:"git-upload-url"`
+	Commit          string `mapstructure:"commit"`
+	ReleaseTemplate string `mapstructure:"release-template"`
```

Comment:
Thought, should this be `TagNameTemplate` to be consistent with `CreateRelease()`?

Since release contains more than just the name/tag name, but the release and tag name should probably match?

Reply:
Good point. The question is whether the focus more is on the tag or the release. I think it's really about creating GitHub releases. The tag is created implicitly. We only deal with release creation.

Reply:
OK, maybe in that case rename to `ReleaseNameTemplate`? Not trying to bikeshed naming, but the release name is only one part of the release (I imagine a future feature allowing release description template, etc)?

Reply:
Makee sense